packages/driver/src/multi-domain/index.ts

```diff
@@ -51,6 +65,12 @@ const setup = () => {
       path: '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome',
       version: '90.0.4430.212',
     },
+    defaultCommandTimeout: 4000,
```

Comment:
Do these needs to be synced with the config in #19590?

Reply:
Yeah, that will be the responsibility of #19590. This is fine for now.